pkg/convert/go_test.go

```diff
@@ -0,0 +1,89 @@
+package convert
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+var modelMsg = model.Messages{
+	Language: language.English,
+	Messages: []model.Message{
+		{
+			ID:          "1",
+			Message:     "message1",
+			Description: "description1",
+			Fuzzy:       true,
+		},
+		{
+			ID:          "2",
+			Message:     "message2",
+			Description: "description2",
+			Fuzzy:       false,
+		},
+	},
+}
+
+func TestToGo(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name    string
+		m       model.Messages
+		want    []byte
+		wantErr bool
+	}{
+		{
+			name:    "When input is correct, then return expected result",
+			m:       modelMsg,
+			want:    []byte(`{"language":"en","messages":[{"id":"1","meaning":"description1","message":"","translation":"message1","fuzzy":true},{"id":"2","meaning":"description2","message":"","translation":"message2"}]}`), //nolint:lll
```

Comment:
Maybe we can export this long []byte to the model object pipeline.Messages and use json.Marshal? Because I think it would be more readable and shorter.

Reply:
It's up to you. Personally, I would use a byte slice.

Reply:
You could do
```go
[]byte(`{
	"language":"lv-lv",
	"messages":[
		 {
				"id":"1",
				"meaning":"When you great someone",
				"message":"hello",
				"translation":"čau",
				"fuzzy":false
		 }
	]
}`,
)
```

but with correct identation